--- a/config.def.h	2021-08-11 11:27:04.937526835 +0300
+++ b/config.def.h	2021-08-11 11:34:03.236289601 +0300
@@ -115,16 +115,22 @@
   [13] = "#e1acbb", /* magenta */
   [14] = "#a7c7a2", /* cyan    */
   [15] = "#e2d3ba", /* white   */
+
+  [255] = 0,
+
+  [256] = "#1a1a1a", /* background */
+  [257] = "#bbbbbb", /* foreground */
+  [258] = "#bbbbbb",
 };
 
 /*
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-unsigned int defaultfg = 15;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 15;
-static unsigned int defaultrcs = 257;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+static unsigned int defaultcs = 257;
+static unsigned int defaultrcs = 256;
 
 /*
  * Default shape of cursor
